### ISSUE TITLE
Uncaught TypeError: Super expression must either be null or a function, not undefined

### ISSUE THREAD

**Issue by j0hny666** [heilhead/react-bootstrap-validation#70]
Hello, 

I have recently updated my modules and started seeing this error. When debugging bundle.js, I have found out that it is caused with ValidatedInput having its parent "undefined".

React v15.3.1
React-bootstrap installed as a dependency
React-bootstrap-validation current version
Babel 6

Thanks for looking into this!
![spectacle gj3115](https://cloud.githubusercontent.com/assets/7735279/17932610/1d7d7c26-6a11-11e6-9735-769b65aba581.png)

**Comment by Serexx** [comment]
Moi aussi ("me too") - webpack dependencies are below. I _really_ do not want to have to switch packages or roll my own but may have to if no solution is readily available... error is identical to that shown by OP.  Any suggestions greatly apreciated

"dependencies": {
    "css-loader": "^0.23.1",
    "extract-text-webpack-plugin": "^1.0.1",
    "flux": "^2.1.1",
    "history": "^3.0.0",
    "postcss-loader": "^0.10.1",
    "react": "^15.3.0",
    "react-dom": "^15.3.0",
    "react-router": "^2.6.1",
    "style-loader": "^0.13.1",
    "react-bootstrap": "^0.30.3",
    "react-bootstrap-validation": "^0.1.11"
  },

**Comment by suman-tms** [comment]
I am also having the same problem..

**Comment by j0hny666** [comment]
I have found out what causes the issue: Input.js was removed from react-bootstrap and replaced by FormControl.js, but ValidatedInput.js still relies on former Input component. Unfortunately, I don't have enough experience to attempt fixing this issue myself, but at least this information can help others.

**Comment by LoicUV** [comment]
Duplicate of #69 #65 #56 ...

**Comment by xizhao** [comment]
New react-bootstrap breaks with this component.

**Comment by salujaharkirat** [comment]
Yeah broken for me too.. @xizhao  can you tell which version of react-bootstrap works with this?

**Comment by xizhao** [comment]
@jinxac Sure, for me anything under react-bootstrap `<0.30.0` works, i'm on `0.29.5`

**Comment by westhomas** [comment]
+1

**Comment by lakhansamani** [comment]
+1

**Comment by sshaginyan** [comment]
+1

**Comment by mikedevita** [comment]
👍

**Comment by dvdgarcia** [comment]
A lot of errors using a lower version :(

**Comment by nguyenbathanh** [comment]
+11

**Comment by rodrigooler** [comment]
+1

**Comment by UpCoder** [comment]
can I use it with react-bootstrap(version > 0.30.0 )?

**Comment by JohnHanna86** [comment]
+1

**Comment by MaxTwentythree** [comment]
+1

**Comment by kristiandelay** [comment]
I have got this to work downgrading react-bootstrap back to ^0.28.1
anything above that seems to break with this error

Working with:
```
...
"react-bootstrap": "^0.28.1",
"react-bootstrap-validation": "^0.1.11",
....
```

**Comment by mikebz** [comment]
it's too bad that downgrading is the only solution

**Comment by nguyenbathanh** [comment]
I suggest using [redux-form](https://github.com/erikras/redux-form). It's very powerful form validation component for react.

**Comment by AlexanderKozhevin** [comment]
+1

**Comment by satenderjoon** [comment]
+1

**Comment by vpste1** [comment]
+1

**Comment by youslydogyou** [comment]
+1

**Comment by PDS42** [comment]
Why do you all say "+1" instead of just thumbs up on the issue? If you have insight that can help resolving it, post your config and screenshots that could help further debugging. Otherwise you're just spamming this thread for no reason. This is likely a babel issue regarding circular module imports. I'm looking into it, will come back with my solution if I solve it.

**Comment by mikedevita** [comment]
> Why do you all say "+1" instead of just thumbs up on the issue? If you have insight that can help resolving it, post your config and screenshots that could help further debugging. Otherwise you're just spamming this thread for no reason.

It goes back to the days before GitHub (GH) had the thumbs up/down reactions, a lot of people still post it out of habit, myself included (often times because I am replying via email on my phone).

I wish GH would have some automated workflow that would take comments with just "+1" or "-1" and convert them to thumbs up/down automatically and delete the comment or hide it.

I agree it is quite spammy.

**Comment by neilyoung** [comment]
+1

**Comment by AdrianMrn** [comment]
Any fix for this yet except for downgrading react-bootstrap, as far as anyone knows?
